ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHeathCard.tsx

```diff
@@ -0,0 +1,117 @@
+    const [isFetching, setIsFetching] = useState(false);
+    const [requestErrorMessage, setRequestErrorMessage] = useState('');
+    const [items, setItems] = useState<IntegrationHealthItem[]>([]);
+
+    useEffect(() => {
+        setIsFetching(true);
+        fetchDeclarativeConfigurationsHealth()
+            .then((itemsFetched) => {
+                setRequestErrorMessage('');
+                setItems(itemsFetched);
+            })
+            .catch((error) => {
+                setRequestErrorMessage(getAxiosErrorMessage(error));
+                setItems([]);
+            })
+            .finally(() => {
+                setIsFetching(false);
+            });
+    }, [pollingCount]);
+
+    /*
+     * Wait for isFetching only until response to the initial request.
+     * Otherwise count temporarily disappears during each subsequent request.
+     */
+    const hasCount = (pollingCount !== 0 || !isFetching) && !requestErrorMessage;
+    const itemCount = items.length;
+
+    return (
+        <Card isFullHeight isCompact>
+            <CardHeader>
+                <CardHeaderMain>
+                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                        <FlexItem>
+                            <CardTitle component="h2">Declarative configuration</CardTitle>
+                        </FlexItem>
+                        {hasCount && (
+                            <FlexItem>
+                                <IconText
+                                    icon={
+                                        itemCount === 0 ? (
+                                            <CheckCircleIcon color="var(--pf-global--success-color--100)" />
+                                        ) : (
+                                            <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+                                        )
+                                    }
+                                    text={
+                                        itemCount === 0
+                                            ? 'no errors'
+                                            : `${itemCount} ${pluralize('error', itemCount)}`
+                                    }
+                                />
+                            </FlexItem>
+                        )}
+                    </Flex>
+                </CardHeaderMain>
+            </CardHeader>
+            {(requestErrorMessage || itemCount !== 0) && (
+                <CardBody>
+                    {requestErrorMessage ? (
+                        <Alert isInline variant="warning" title={requestErrorMessage} />
+                    ) : (
+                        <TableComposable variant="compact">
+                            <Thead>
+                                <Tr>
+                                    <Th width={40}>Name</Th>
+                                    <Th width={40}>Error</Th>
```

Comment:
I'll conduct some experiments on the real data to see if having 40/40/20 split makes sense - perhaps something closer to 30/55/15 would work better

Reply:
Thank you. I usually test at 1920px panel width and 1440px laptop width (by opening developer panel at the right and dragging edge until desired page width appears in screen tip).

Let’s also think ahead to the next step when whatever percent for Name in declarative configuration will correspond to the sum of Name and Type for integrations (that is, so Error and Date align consistently if more than one card has a table).